Kitodo/src/main/java/org/goobi/production/cli/helper/CopyProcess.java

```diff
@@ -68,7 +69,7 @@
 
 public class CopyProcess extends ProzesskopieForm {
 
-    private static final Logger myLogger = Logger.getLogger(ProzesskopieForm.class);
+    private static final Logger logger = LogManager.getLogger(ProzesskopieForm.class);
```

Comment:
Can you please correct the used class name from `Prozesskopieform.class` to `CopyProcess.class` as the class name is `CopyProcess`?